fury/window.py

```diff
@@ -563,17 +580,27 @@ def __init__(
         qt_parent : QWidget, optional
             An existing QWidget to embed the QtCanvas within (if `window_type`
             is 'qt').
+        show_fps : bool, optional
+            Whether to display FPS statistics in the renderer.
+        max_fps : int, optional
+            Maximum frames per second for the canvas.
         """
         self._size = size
         self._title = title
         self._is_qt = False
         self._qt_app = qt_app
         self._qt_parent = qt_parent
         self._is_initial_resize = None
+        self._show_fps = show_fps
+        self._max_fps = max_fps
         self._window_type = self._setup_window(window_type)
 
+        # For offscreen rendering, enable console FPS printing in the renderer
+        # For interactive windows, we'll use visual Stats overlay instead
+        use_renderer_fps = self._show_fps and self._window_type == "offscreen"
+
         if renderer is None:
-            renderer = Renderer(self.window)
+            renderer = Renderer(self.window, show_fps=use_renderer_fps)
```

Comment:
`use_renderer_fps` does not make sense as we will not be able to see it. The moment I move it offscreen it will open and shut. You will not see any print on the terminal.

Let's not do anything in the case of `offscreen window_type`.